LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2016-2023 zOS Global Limited and contributors
```

Comment:
This should say 2024 & ScopeLift

Reply:
Good catch, updated [f0070f1](https://github.com/ScopeLift/stealth-address-erc-contracts/pull/8/commits/f0070f1da6a984adb31929d51678fa53169a4eb5)